tests/testKinematicsPhase.cpp

```diff
@@ -25,7 +25,6 @@ using std::string;
 #include "contactGoalsExample.h"
 
 TEST(Phase, InverseKinematics) {
-  // TODO(frank): create test for Phase, derived from interval?
```

Comment:
Empty test? Perhaps put in a comment detailing what the intent for this test is, else kill it?